lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -50,3 +50,7 @@ export enum TemporaryCacheKeys {
     REQUEST_PARAMS = "request.params",
     SCOPES = "scopes"
 }
+
+export const DEFAULT_REQUEST = {
+    scopes: ["openid", "profile"]
```

Comment:
Use the constant vars instead of strings here.

Reply:
`Constants.OPENID_SCOPE, Constants.PROFILE_SCOPE`

Reply:
Yep good call! Done.